tests/test_import_entry_points.py

```diff
@@ -43,5 +43,36 @@ def do_import(module: str) -> Optional[int]:
     yield do_import
 
 
+def test_import_exceptions(import_entry_point) -> None:
+    assert import_entry_point("inmanta.ast") == 0
+    assert import_entry_point("inmanta.parser") == 0
+
```

Comment:
Also test import of:

* `inmanta.export`: This is required if you want use the `@dependency_manager` annotation
* `inmanta.execute.util`: This is required if you want to return a `Unknown`

Reply:
I've also added these to the reference API documentation.